lms/djangoapps/verify_student/services.py

```diff
@@ -136,3 +137,25 @@ def get_attempts(self, user_id, course_id, related_assessment_location):
         """
         course_key = CourseKey.from_string(course_id)
         return VerificationStatus.get_user_attempts(user_id, course_key, related_assessment_location)
+
+    def get_course_verification_status(self, user, course_id):
```

Comment:
@nedbat both CourseEnrollment.get_enrollment() and check_verify_status_by_course() expect a User object. I can certainly change this call signature here to be user_id and then do a lookup, but it seems unnecessary to have to do another database round trip.